### ISSUE TITLE
Design a PCB that is RFI resistant for remote location close to high power antennas

### ISSUE THREAD

**Issue by K7MDL2** [K7MDL2/RF-Power-Meter-V1#10]
A version built on proto PCB with lots of short wire connection in a plastic case, located within 25ft of the several VHF antennas often locks up when transmitting with high power levels like 150W to 500W.  Several remedies have help a bit, but it is time to create a PCB with particular attention to RFI resistance,  To that end I have a nearly completed PCB waiting for parts to arrive and do a part fit verification before ordering boards.

**Comment by K7MDL2** [comment]
Created an optocoupler based board with only ethernet cable inside on 3/31/2022. Ordered 8 boards and built 1 of them.    Made several tweaks to the board as I built the first one including routing the USB host and ethernet cables wired onto the board and fixed a missing 3.3V trace.

**Comment by K7MDL2** [comment]
Testing completed.  Found an instance of 1 short 3.3V trace crossing over 2 signal traces.  The 3.3V line is not used so is cut away from the 2 signal traces that feed the Teensy onboard ADC inputs.   I have made several improvements to a new PCB version as I built this first one, but the 1st boards are totally usable as is. The ethernet 6 pin connector was 1mm pitch instead of the needed 2mm pitch so I soldered the ribbon cable wires from the Teensy into the holes and ethernet nest is working.
The old CPU and ADL5519 detector board was installed preserving the calibration for the most part.  Each IO line was tested.  

Tonight I placed it into service with out a case adjusting for changes in IO pin inversions on the input and outputs relative the old unit.  The old unit would corrupt the CPU at about 100W on 2M, this PCB version handled 500W for a few hours for the Spring VHF 2M sprint. NO problem at 500W on 432 or any other band.  

Declaring this a victory and now will prepare metal box front an back panels to finish this up.  Preparing a PowerPoint also.  Tested with a 3.5" Nextion display and using a 0.96" OLED.

I plan to order some updated PCBs once the existing supply of 7 are exhausted.
